src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.LogicManager.FILE_OPS_ERROR_FORMAT;
+import static seedu.address.logic.LogicManager.FILE_OPS_PERMISSION_ERROR_FORMAT;
+
+import java.io.IOException;
+import java.nio.file.AccessDeniedException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.address.commons.util.FileUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.storage.Storage;
+
+/**
+ * Exports the contacts to a file akin to the save file.
+ */
+public class ExportCommand extends FileAccessCommand {
+    public static final String COMMAND_WORD = "export";
+    public static final String MESSAGE_SUCCESS = "Address book has been exported!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+        + ": Exports the current contacts to a json file with its "
+        + "name being the current time and date. ";
+    private static final DateTimeFormatter EXPORT_FORMATTER = DateTimeFormatter.ofPattern("MM-dd-yyyy-hhmmssa");
+
+    @Override
+    public CommandResult execute(Model model) {
+        // This version of execute should never be run
+        assert false;
+        requireNotExecuted();
+        requireNonNull(model);
+        isExecuted = true;
+        return new CommandResult(MESSAGE_SUCCESS);
```

Comment:
Instead of returning a success command result, maybe it is better to not have any statements in this method, and just throw an Assertion error directly?

Reply:
I don't feel that will fit the convention all the other command returns success message and it can also be used for testing, I advocate for keep the message